spacy/language.py

```diff
@@ -1149,13 +1472,15 @@ def _fix_pretrained_vectors_name(nlp):
 class DisabledPipes(list):
     """Manager for temporary pipeline disabling."""
 
-    def __init__(self, nlp, names):
+    def __init__(self, nlp: Language, names: List[str]):
         self.nlp = nlp
         self.names = names
         # Important! Not deep copy -- we just want the container (but we also
         # want to support people providing arbitrarily typed nlp.pipeline
         # objects.)
         self.original_pipeline = copy(nlp.pipeline)
+        self.metas = {name: nlp.get_pipe_meta(name) for name in names}
+        self.configs = {name: nlp.get_pipe_config(name) for name in names}
         list.__init__(self)
         self.extend(nlp.remove_pipe(name) for name in names)
```

Comment:
Does `nlp.remove_pipe` change the config?

I think either answer will be a bit painful. If it does, it will be a rare time when the config updates from state changes, which will be a bit inconsistent.

But we do need to change the config here, don't we? `nlp.add_pipe` changes the config so that we load back in the updated pipeline. And disable pipes is how we've asked people to change the pipeline before saving previously.